### ISSUE TITLE
[salt-cloud lxc] salt-cloud doesn't see existing machines offers to create them

### ISSUE THREAD

**Issue by arthurlogilab** [saltstack/salt#23659]
After creating lxc machines using salt-cloud  (and fixing them as mentioned in #23657 ). Re-running salt-cloud -m map shows them as running in the (too) verbose log output, but still considers them as non existent and offers to create them.

**Comment by jfindlay** [comment]
@arthurlogilab, is it possible for you to share your cloud map file?  Thanks.

**Comment by arthurlogilab** [comment]
nothing fancy : 

```
wheezy_ursa_lxc:
  - host1.logilab.priv
jessie_ursa_lxc:
  - host2.logilab.priv
  - host3.logilab.priv
```

**Comment by rallytime** [comment]
This should be fixed by #23770, which was available starting with 2015.5.1. As such, I am going to close this. If this pops up again, let us know and we can take another look. Thanks!
